pkg/ratelimit/runner.go

```diff
@@ -42,16 +42,16 @@ const (
 
 // Runner is the interface for running tasks.
 type Runner interface {
-	RunTask(ctx context.Context, name string, f func(context.Context), opts ...TaskOption) error
+	RunTask(id, name string, f func(), opts ...TaskOption) error
```

Comment:
I think `RunTask(id uint64, name string, f func(), opts ...TaskOption) error` is ok. `fmt.Sprint` has an allocation cost.

Reply:
But the same id may have different tasks. If we don't use fmt.Sprint, then we need a map[uint64]map[string]struct.

Reply:
If the key is different, the bench will be different